envoy/http/filter.h

```diff
@@ -758,13 +758,14 @@ class StreamDecoderFilterCallbacks : public virtual StreamFilterCallbacks,
    * host list of the routed cluster, the host should be selected first.
    * @param host The override host address.
    */
-  virtual void setUpstreamOverrideHost(absl::string_view host) PURE;
+  virtual void setUpstreamOverrideHost(Upstream::LoadBalancerContext::OverrideHost&&) PURE;
```

Comment:
does this move semantics make sense? The override host just a pair of string view and bool.

Reply:
I think you are right. Move will most likely apply only to std::pair internals. string_view and bool will just copy their values. I will change it to pass by value. Thanks!